script/CompoundGovernorConstants.sol

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: MIT
+// slither-disable-start reentrancy-benign
+pragma solidity 0.8.26;
+
+contract CompoundGovernorConstants {
+    // These constants are taken from the existing GovernorBravoDelegate contract.
+
+    uint48 INITIAL_VOTING_DELAY = 13_140; // The delay before voting takes place, in blocks
```

Comment:
hey @jferas how confident are we that these are the constants we're going live with? should we put a TODO here (or better, an issue to verify these initialization params)?